frontend/src/pages/HomePage.js

```diff
@@ -1,8 +1,21 @@
+import React, { useState, useEffect } from 'react';
 import Product from '../components/Product';
-import products from '../products';
 import MainContent from '../components/MainContent';
+import axios from 'axios';
 
 const HomePage = () => {
+  const [products, setProducts] = useState([]);
+
+  useEffect(() => {
+    const fetchProducts = async () => {
+      const { data } = await axios.get('api/v1/products');
```

Comment:
If you use the `src/api.js` module, then this would become:

```js
const { data } = await axios.get(api.products);
```

Reply:
Awesome..yea wasn't sure where to put those lol. Nice. Learned something new. Thanks